frontend/src/common/types/podcast/podcast-search-payload.type.ts

```diff
@@ -0,0 +1,5 @@
+type PodcastSearchPayload = {
```

Comment:
we already have this type in the shared folder
we need to for the both side, frontend & backend
remove it from this and use that from the shared folder